.github/workflows/CICD.yml

```diff
@@ -149,6 +149,13 @@ jobs:
         ## Run it
         cd fuzz
         cargo +nightly fuzz run fuzz_test -- -max_total_time=${{ env.RUN_FOR }} -detect_leaks=0
+    - name: Run fuzz_expr for XX seconds
+      continue-on-error: true
```

Comment:
for now, it is failing but we don't fail the job on it. 
I opened :
https://github.com/uutils/coreutils/issues/5315
https://github.com/uutils/coreutils/issues/5316

but we probably have more issues